src/Tickets/Commerce/Provider.php

```diff
@@ -28,7 +28,7 @@ class Provider extends Service_Provider {
 	 */
 	public function register() {
 		// Specifically prevents anything else from loading.
-		if ( ! tec_tickets_commerce_is_enabled() ) {
+		if ( ! \tec_tickets_commerce_is_enabled() ) {
```

Comment:
do we still need the `\`. I don't think the fails earlier were related to this one.